src/npc_enchanter.cpp

```diff
@@ -1115,46 +1129,51 @@ class npc_enchantment : public CreatureScript
         if (roll > 0 && roll < 33)
         {
             player->GetSession()->SendNotification("|cff00ff00Beauregard's bony finger crackles with energy when he touches |cffDA70D6%s|cff00ff00!", item->GetTemplate()->Name1.c_str());
-            creature->HandleEmoteCommand(EMOTE_ONESHOT_POINT);
-            player->PlayerTalkClass->SendCloseGossip();
+            // creature->HandleEmoteCommand(EMOTE_ONESHOT_POINT);
         }
         else if (roll > 33 && roll < 75)
         {
             player->GetSession()->SendNotification("|cff00ff00Beauregard holds |cffDA70D6%s |cff00ff00up in the air and utters a strange incantation!", item->GetTemplate()->Name1.c_str());
-            creature->HandleEmoteCommand(EMOTE_ONESHOT_CHEER);
-            player->PlayerTalkClass->SendCloseGossip();
+            // creature->HandleEmoteCommand(EMOTE_ONESHOT_CHEER);
```

Comment:
Instead of commenting on this line, wouldn't it be better to delete it if it is not going to be used? Or failing that, create an option in the conf, for those who want to use it and which is disabled by default.